README.md

```diff
@@ -34,15 +34,15 @@ Why?
 1. Install [asdf-vm](https://asdf-vm.github.io/asdf/#/core-manage-asdf-vm) for your OS.
 2. Using asdf-vm, install [ruby](https://github.com/asdf-vm/asdf-ruby).
 3. Install Jekyll
-    
+
```

Comment:
Reason for this change?

Reply:
I was trying to resolve the issues that the Travis Log said. I thought it failed at indentations, so I removed them. I didn't know there is another issue for this. I'll uncommit it. :)